lintly/parsers.py

```diff
@@ -200,18 +202,27 @@ def parse_violations(self, output):
 
 
 class BlackParser(BaseLintParser):
-    """A parser for the `black [source] --check` command."""
+    """A parser for the `black [source] --diff --check` command."""
+
+    MESSAGE_FORMAT = (
```

Comment:
This lets you make an expandable comment, like this!

---
This code block needs to be reformatted.

<details><summary>Expand to show diff</summary>

```
# giant block of diffs go here!
```
</details>

Reply:
super cool - can't wait to see this in action

Reply:
Here's a preview: https://github.com/23andme-private/sarahc/pull/27/files

(Still trying to figure out why it isn't trigger on check_files.py, though.)